.github/workflows/prune-images.yml

```diff
@@ -1,23 +1,28 @@
 name: Prune Images
-run-name: Prune old images from OpenShift
+run-name: Pruning all but the most recent ${{ inputs.minTags }} tags prior to what is deployed in ${{ inputs.environment }}
 
 on:
   workflow_dispatch:
     inputs:
+      environment:
+        type: choice
+        description: 'Environment'
+        required: true
+        options:
+          - dev
+          - test
```

Comment:
As per our discussion, we are using the reference of build deployed to test. Do we still need this environment input?

Reply:
The default is test but we have flexibility by keeping this which the underlying scripts can use.

Reply:
@dheepak-aot I believe that it would be ok keeping it as it is considering that the cleanup would happen only for the "main-*" builds for now.